parflow_data_management/transport/consumers.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from channels.generic.websocket import JsonWebsocketConsumer
+from asgiref.sync import async_to_sync
+
+class KeyConsumer(JsonWebsocketConsumer):
+    def connect(self):
+        self.accept()
+        group = compute_group_for_user(self.scope["user"].id)
+        async_to_sync(self.channel_layer.group_add)(group, self.channel_name)
+
+    def disconnect(self, close_code):
+        group = compute_group_for_user(self.scope["user"].id)
+        async_to_sync(self.channel_layer.group_discard)(group, self.channel_name)
+
+    def receive(self, text_data):
+        pass
+
+    def public_key_with_passphrase(self, message):
+        self.send_json({
+            "type": "websocket.send",
```

Comment:
Should the `type` not be related to the payload?

Reply:
No, I'll fix that as well.

Reply:
Fixed in 673a16e656b460a42aec36abbc5693cc33ae0684